CONTRIBUTING.md

```diff
@@ -105,7 +105,13 @@ When building with make, cargo will automatically use [pipelined][p] compilation
 
 ## Running TiKV
 
-To run TiKV as an actual key-value store, you will need to run it as a cluster (a cluster can have just one node, which is useful for testing). You can do this on a single machine or on multiple machines. You need to use [PD](https://github.com/tikv/pd) to manage the cluster (even if there is just one node on a single machine). Instructions are in our [docs](https://tikv.org/docs/dev/tasks/deploy/binary/) (if you build TiKV from source, then you don't need to download the binary).
+To run TiKV as an actual key-value store, you will need to run it as a cluster (a cluster can have just one node, which is useful for testing). You can do this on a single machine or on multiple machines. 
+
+Install [PD](https://github.com/tikv/pd) to manage the cluster (even if just one node on a single machine). 
```

Comment:
May I ask why you change the wording to "install"? "Install" sounds like a strong instruction but we don't actually care about it.